RELEASENOTES.md

```diff
@@ -7,6 +7,50 @@ This document contains the release notes for `rippled`, the reference server imp
  
 Have new ideas? Need help with setting up your node? Come visit us [here](https://github.com/xrplf/rippled/issues/new/choose)
 
+# Introducing XRP Ledger version 1.9.4
+
+Version 1.9.4 of `rippled`, the reference implementation of the XRP Ledger protocol is now available. This releases introduces an amendment that removes the ability for an NFT issuer to indicate that trustlines should be automatically created for royalty payments from secondary sales of NFTs, in response to a bug report that indicated how this functionality could be abused to mount a ddenial of service attack against the issuer.
```

Comment:
```suggestion
Version 1.9.4 of `rippled`, the reference implementation of the XRP Ledger protocol is now available. This releases introduces an amendment that removes the ability for an NFT issuer to indicate that trust lines should be automatically created for royalty payments from secondary sales of NFTs, in response to a bug report that indicated how this functionality could be abused to mount a denial of service attack against the issuer.
```

Reply:
I've been calling it a trustline for a decade, and a lot of the documentation does as well, but I'm ok with changing this.